README.md

```diff
@@ -144,6 +144,11 @@ else
 end
 ```
 
+Issues
+------
+Please fill out our [issues template](.github/issue_template.md) if you are
+having problems.
+
```

Comment:
This could probably be deleted. GitHub does it automagically! https://github.com/thoughtbot/parity/issues/new